mapserver/mapfiles/wms.map/layers/hurricane_noaa_forecast_layer.layer.jinja2

```diff
@@ -0,0 +1,48 @@
+
+    LAYER
+		NAME "hurricane_noaa_forecast_layer"
+		METADATA
+			"ows_title" "hurricane_noaa_forecast_layer"
+		END
+		STATUS ON
+		TYPE POLYGON
+		CONNECTIONTYPE postgis
+		CONNECTION "host={{ POSTGRES_HOST }} port={{ POSTGRES_PORT }} dbname={{ POSTGRES_DB }} user={{ POSTGRES_USER }} password={{ POSTGRES_PASS }}"
+		DATA "geometry from (select
+            row_number() OVER () AS gid,
+            hazard_event.id as id,
+            geometry,
+            depth_class
+            from hazard_event
+            inner join hazard_map on hazard_event.flood_map_id = hazard_map.id
+            inner join hazard_areas on hazard_map.id = hazard_areas.flood_map_id
+            inner join hazard_area on hazard_areas.flooded_area_id = hazard_area.id) as subquery
+         using unique gid using srid=4326"
+
+		CLASS
+			NAME "PTC"
+			EXPRESSION ([depth_class] == 83)
+			STYLE
+				OPACITY 30.0
+				COLOR "#fbfbfb"
+			END
+		END
+
+		CLASS
+			NAME "Subtropical Storm"
+			EXPRESSION ([depth_class] == 84)
+			STYLE
+				OPACITY 30.0
+				COLOR "#fda7a7"
+			END
+		END
+
+		CLASS
+			NAME "Subtropical Depression"
+			EXPRESSION ([depth_class] == 85)
+			STYLE
+				OPACITY 30.0
+				COLOR "#ff0000"
+			END
+		END
```

Comment:
I am not really sure this is the hazard class.
It just seems to be a storm type, not indicative of damage.